src/main/java/hudson/plugins/jira/JiraGlobalConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package hudson.plugins.jira;
+
+import hudson.Extension;
+import hudson.util.PersistedList;
+import java.util.List;
+import javax.annotation.Nonnull;
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.Jenkins;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundSetter;
+
+@Extension
+@Symbol("jira")
```

Comment:
A bit too generic name IMHO.

Reply:
I can remove the symbol and then it would just be 
```yaml
unclassified:
  jiraglobalconfiguration:
    sites:
    - url: https://jira.com/
```